libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/OAuth2.java

```diff
@@ -106,7 +105,7 @@ public class OAuth2 {
     private static final String PHOTOS = "photos";
     private static final String PICTURE = "picture";
     private static final String THUMBNAIL = "thumbnail";
-    private static final String AUTHORIZATION_CODE = "authorization_code";
+    private static final String AUTHORIZATION_CODE = "hybrid_auth_code";
```

Comment:
Using hybrid auth code flow instead of regular auth code when exchanging code for tokens.